### DISCUSSION TITLE
The audio command supports input and output but not Sound Effects

### DISCUSSION THREAD

**Discussion by rickyrodmac** [ttscoff/bunch-docs#214] [Ideas]
I would like to set the `Sound Effects` to play through the `MacBook Pro Speakers` and the other sounds through my various bluetooth headphones.

Currently setting the output to my headphones works great but the system sounds interrupt conf. calls. 

I have to manually set `Play sound effects through` after the bunch sets my desired headphones.

**Comment by ttscoff** [comment]
I'm not certain this is possible with the APIs I'm using, for some 
reason, but I'll look into it again.

-Brett

**Comment by rickyrodmac** [comment]
Is this feature request reasonable? The only tool I was able to find that does this is SoundSource from Rogue Amoeba. I'm hoping that I could save some money. 😄

**Comment by ttscoff** [comment]
First off, SoundSource is worth every penny, but that's just my opinion :).

Yes, the request is entirely reasonable, just have to see if that's something I have access to via the API I'm using. There was some reason I didn't include it initially but I need to review to remember why.
